Gemfile

```diff
@@ -10,6 +10,7 @@ gemspec
 # Note that if allow_local is true, but the gem is not found locally, then it will
 # checkout the latest version (develop) from github.
 allow_local = ENV['FAVOR_LOCAL_GEMS']
+allow_local = 1
```

Comment:
Can we remove these hard-coded variables in this Gemfile and example_files/Gemfile? Obviously helpful while developing, but should be removed for the final version.